src/initialization/FGLinearization.cpp

```diff
@@ -20,23 +20,19 @@
 #include "FGLinearization.h"
 #include <ctime>
 
-namespace JSBSim {
 
-// TODO make FGLinearization have X,U,Y selectable by xml config file
+namespace JSBSim {
 
-FGLinearization::FGLinearization(FGFDMExec * fdm, int mode)
+FGLinearization::FGLinearization(FGFDMExec * fdm) : aircraft_name(fdm->GetAircraft()->GetAircraftName())
 {
-    std::cout << "\nlinearization: " << std::endl;
-    std::clock_t time_start=clock(), time_linDone;
-    FGStateSpace ss(fdm);
-
+    auto ss = FGStateSpace(fdm);
```

Comment:
I originally changed it so that the FGStateSpace instance was a member of the class so that FGLinearization can access it later on. This was eventually not necessary so I changed it "back". I didn't check what was originally there when I started.

Reply:
No problem. That's what code review are made for 😉